packages/amplify-cli-core/src/state-manager/stateManager.ts

```diff
@@ -263,6 +268,64 @@ export class StateManager {
     });
   };
 
+  getResourceFromMeta = (
+    amplifyMeta: Record<string, any>,
+    categoryName: string,
+    serviceName: string,
+    resourceName?: string | undefined,
```

Comment:
As it is not the last argument undefined is needed to be specified as cannot be optional without it, it could be removed but makes no harm or difference.